src/main/java/com/google/sps/data/UserAccount.java

```diff
@@ -4,8 +4,6 @@
 
 public class UserAccount {
 
-  public static final String ENTITY_TYPE = "UserAccount";
-
   private String nickname;
```

Comment:
Do you also plan to have a name, or is nickname the same as name here?

Reply:
The nickname is the same as name here, we just named the field 'nickname' since it's collected as the screen name from the users instead of legal name.